### PR TITLE
[DOCS-6396] Align Support Handbook to Hyland

### PR DESCRIPTION
**Pull request by anxumalo**
Initial updates to the Alfresco Support Handbook to align support delivery to Hyland’s standard model. More changes to follow.

### CODE REVIEW

File: support/latest/severity.md [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
-|4 - Question - How To - Enhancement|A Severity Four issue is a general usage question, request for information, reporting of a documentation error, or recommendation for a future product enhancement or modification.  There is low-to-no impact on your business or the performance or functionality of your system.|
+The outlined Hyland Support severity definitions are utilized when prioritizing and working on your issue. Reserve communication of Severity Level One (1) for your most critical issues (production environment down) to ensure the Hyland Support team can respond to all issues appropriately.
+
+| Severity | Definition |
+|--------- | ---------- |
+| Level 1 | "Level 1" means any Error that causes total or substantial software failure, which means that the software is down and the customer is unable to access the software in any way within their production environment. |
+| Level 2 | "Level 2" means an Error that causes substantial software failure which prevents a portion of a customer’s users from accessing the software in any way within the production environment. |
```
Comment (oliveralfresco2018) [inline support/latest/severity.md:10]: The numbering appears weird on mine. Level 1 is ok but Level 2 onwards the 2 goes onto the line underneath.

File: support/latest/slt.md [hunk 2]
```diff
@@ -2,26 +2,17 @@
 title: Service Level Targets (SLTs)
 ---
 
-You can view your support cases at any time via the Support Portal. Below are the target response times for a support case based on the customer's subscription type and support case severity level.
+You can open a new support case or view your historical support cases at any time via [Community](https://community.hyland.com/){:target="_blank"} on the Open Support Cases page. Below are the target response times for a support case based assigned severity level.
```
Comment (oliveralfresco2018) [inline support/latest/slt.md:5]:
```suggestion
You can open a new support case or view your historical support cases at any time via [Community](https://community.hyland.com/){:target="_blank"} on the Open Support Cases page. Below are the target response times for a support case based on assigned severity level.
```

File: support/latest/slt.md [hunk 3]
```diff
@@ -2,26 +2,17 @@
...
-|Three (3)|Production/Development - Minor Impact|Every two business days|Every two business days|Every three business days|
-|Four (4)|Question/How-to/Enhancement|Every four business days|Every four business days|Every five business days|
+| Severity | Hyland Response |
+| -------- | --------------- |
+| Level 1 | Upon receiving notification from a customer, a Hyland support Manager is immediately notified.  Within thirty (30) minutes, the Manager will notify a member of Support Senior management.<br><br>If there is no resolution within two (2) hours of a customer’s notice, Hyland will place the customer on the High Visibility Ticker (HVT).<br><br>If there is no resolution within four (4) hours of the customer’s notice or by the end of business of that day, Hyland will designate the Error as Code Blue.  Designation as Code Blue means a resolution team is immediately formed for the Error and the resolution team provides continuous updates to the C-Level Executives and Vice Presidents of Hyland.<br><br>To provide a resolution, Hyland will match the customer’s effort, up to and including 24 hours days, 7 days a week, through holidays and weekends until there is a resolution. |
```
Comment (oliveralfresco2018) [inline support/latest/slt.md:13]: the numbering here goes onto the line underneath also like I described above. cheers.

**Review by oliveralfresco2018 — COMMENTED** [review]
All looks ok except a few minor bits.

File: support/latest/severity.md (hunk 1 above)
Comment (anxumalo) [reply support/latest/severity.md:10]: I don' think we can fix this

File: support/latest/slt.md (hunk 3 above)
Comment (anxumalo) [reply support/latest/slt.md:13]: I don' think we can fix this

File: support/latest/packages.md [hunk 4]
```diff
@@ -2,78 +2,64 @@
...
+| Assist with Web Client and Server Configuration |  | Y |
+| Assist with User Admin guidance, and role and workflow configuration |  | Y |
+| Assist with common customizations |  | Y |
+| Basic Setup and Installation |  | Y |
+| Annual Performance Tuning |  | Y |
+| Customer Council Candidate(s) |  | Y |
+
+## Standard Support
+
+Ideal for high-availability systems, Standard Support offers fast response times. Any production system down issue (Severity Level 1) is responded to on a 24x7 basis. Our Analysts work with your team to address the issue in the shortest possible time. Any other severity issues are initially responded to within the targeted service level response time (see the Target Service Level Target section).
```
[the remark this answers is not in the document]
Comment (anxumalo) [reply support/latest/packages.md:38]: Fixed - will link to `slt.md` page - reworded to say "Service Level Targets (SLTs)"

File: support/latest/packages.md [hunk 5]
```diff
@@ -2,78 +2,64 @@
...
+| 24x7 Support | Y | Y |
+| Alfresco University - Passport |  | Y |
+| Premier Services Engineer |  | Y |
+| Release Upgrade Assistance |  | Y |
+| Weekly Case Reviews |  | Y |
+| Quarterly Service Reviews |  | Y |
+| Hot Fix Build Priority Escalation |  | Y |
+| Annual Health Check |  | Y |
+| Assist with Web Client and Server Configuration |  | Y |
+| Assist with User Admin guidance, and role and workflow configuration |  | Y |
```
[the remark this answers is not in the document]
Comment (anxumalo) [reply support/latest/packages.md:30]: Hmm .. good question! It sounds like the last two things are grouped together, so making an executive decision & going with "Assist with User Admin guidance, plus role and workflow configuration" :-)

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM cheers

**Pull request merged**